django/tasks/base.py

```
@@ -0,0 +1,83 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.module_loading import import_string
+
+from .exceptions import InvalidTaskBackendError
+
+DEFAULT_TASK_ALIAS = 'default'
+
+# backends cache to make sure we don't create one per connection
+# all the backends should be thread-safe
+backends = {}
+
+
+def _create_task_backend(alias):
+    try:
+        conf = settings.QUEUES[alias]
+    except KeyError:
+        if alias != DEFAULT_TASK_ALIAS:
```

I don't understand the purpose of this condition. Do we really want to silently return the default backend if any nonexistent alias is given?